pkg/apis/cosigned/v1alpha1/clusterimagepolicy_defaults.go

```diff
@@ -14,8 +14,20 @@
 
 package v1alpha1
 
-import "context"
+import (
+	"context"
+	"fmt"
+)
 
 // SetDefaults implements apis.Defaultable
-func (*ClusterImagePolicy) SetDefaults(ctx context.Context) {
+func (c *ClusterImagePolicy) SetDefaults(ctx context.Context) {
+	c.Spec.SetDefaults(ctx)
+}
+
+func (spec *ClusterImagePolicySpec) SetDefaults(ctx context.Context) {
+	for i, authority := range spec.Authorities {
+		if authority.Name == "" {
+			spec.Authorities[i].Name = fmt.Sprintf("authority-%d", i)
```

Comment:
Same question...  `attestation` seems weird.

Reply:
lol, I originally had it as attestation, thought it seemed wierd, so changed it to authority-%d like here, but it's correct here? Or how do you mean? Above, yes I had neglected to update the comment, but the code is right here, no?